crates/amaru-ledger/src/rules/transaction/inputs.rs

```diff
@@ -18,14 +18,14 @@ use thiserror::Error;
 
 #[derive(Debug, Error)]
 pub enum InvalidInputs {
-    #[error("Unknown input: {}#{}", .0.index, .0.transaction_id)]
+    #[error("Unknown input: {}#{}", .0.transaction_id, .0.index)]
```

Comment:
:'( 

I know this is how cardano-cli typically shows output reference, but it's a terrible convention IMO. Having the index far at the back makes it easy to miss. But, I won't fight for it (now).